cabal.project

```diff
@@ -54,6 +54,10 @@ package bitvec
 package plutus-scripts-bench
   haddock-options: "--optghc=-fplugin-opt PlutusTx.Plugin:defer-errors"
 
+constraints:
+  io-classes-mtl < 0.1.2.0,
```

Comment:
What's the status on `io-classes-mtl`?

Reply:
https://github.com/input-output-hk/io-sim/issues/164